inference-engine/src/mkldnn_plugin/memory_desc/dnnl_blocked_memory_desc.cpp

```diff
@@ -9,9 +9,18 @@
 using namespace MKLDNNPlugin;
 using namespace InferenceEngine;
 
-DnnlBlockedMemoryDesc::DnnlBlockedMemoryDesc(InferenceEngine::Precision prc, const Shape& shape) : MemoryDesc(shape, DnnlBlocked) {
+DnnlBlockedMemoryDesc::DnnlBlockedMemoryDesc(InferenceEngine::Precision prc, const Shape& shape, const VectorDims& strides)
+    : MemoryDesc(shape, DnnlBlocked) {
     const auto ndims = shape.getRank();
     const auto &dims = shape.getDims();
+
+    if (!strides.empty()) { // custom strides
+        desc = {MKLDNNExtensionUtils::convertToDnnlDims(dims),
+                MKLDNNExtensionUtils::IEPrecisionToDataType(prc),
+                MKLDNNExtensionUtils::convertToDnnlDims(strides)};
+        return;
+    }
```

Comment:
In such case neither order nor blokced params are initialized. Probably we should add just another branch into the strides initialization logic

Reply:
Done